src/org/zaproxy/zap/extension/websocket/WebSocketProxy.java

```diff
@@ -186,6 +186,11 @@ public int getIntValue() {
 	 */
 	private boolean isClientMode;
 	
+	/**
+	 * In server mode there is no connection ZAP <-> Server, but only Browser <-> ZAP.
+	 */
+	private boolean isServerMode;
+	
```

Comment:
Does this require a new variable? Can client and server be true at the same time?

Reply:
I think it's needed, we have 3 "modes" now (client, proxy, and server), but could use an enum internally instead of two booleans?

Reply:
Doesn’t matter to me too much, just looked funny when I saw them side by side

Reply:
I'm good with an enum, makes it a bit clearer